.github/workflows/ci.yml

```diff
@@ -26,8 +63,38 @@ jobs:
         with:
           check-latest: true
           go-version-file: "go.mod"
+          cache: ${{ runner.arch != 'arm64' }}
```

Comment:
Why?

Reply:
our selfhosted runners don't need caching - they keep the downloaded modules between builds anyway. But their cache is huge (several GiB) and this slows down the build -> the cache needs to be uploaded and downloaded during every build. Not caching speeds up builds significantly on our self hosted runners.